examples/basic-with-tests-cucumber/docker/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+# see https://www.digitalocean.com/community/tutorials/how-to-share-data-between-docker-containers
+# https://docs.docker.com/compose/reference/
+services:
+  server:
```

Comment:
If you change the service name `server` to match the name of the local project being built `basic-cucumber` you will get rid of the following error in the output from `dockerComposeUp`:

`Error response from daemon: repository basic-cucumber not found: does not exist or no pull access`

The plugin attempts to find which service in your docker compose file is being built locally and will not attempt to download it. It does this by attempting to match the name of the sbt Project to one of the service names in the Docker Compose file.